README.md

```diff
@@ -88,9 +88,8 @@ samples will be kept up to date: use `.. testcode::` and `.. testoutput::` rathe
 
 1. Developer XX works on his branch (XX-...).
 When finished, he completes the RELEASE.md without writing the version number (he completes under ## next).
-He then creates a pull request into master.
-2. Once the pull request has been accepted by an administrator (tests must pass, among other things), the branch is merged on master.
-3. When administrator wants to create a version, he completes RELEASE.md with version number, sets it version.py, commits and creates a tag (vX.X.X).
+He then creates a pull request into develop.
+2. Once the pull request has been accepted by an administrator (tests must pass, among other things), the branch is merged on develop.
+3. When administrator wants to create a version, on branch "develop" he completes RELEASE.md with version number, commits and creates a tag (vX.X.X).
```

Comment:
no.

2 cases, concerning RELEASE.md :
* new version is called "next" -> the tests will be run, but not the versioning
* new version is called "x.x.x" -> the tests will be run, and the versionning too (package + pypi + conda)